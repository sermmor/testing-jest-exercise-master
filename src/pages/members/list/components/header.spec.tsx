import * as React from 'react';
import { Header } from './header';
import { shallow } from 'enzyme';

describe('pages/members/list/components/header test', () => {
    it('should be a valid header rendered', () => {
        const props = {};

        const component = shallow(<Header/>);

        expect(component.childAt(0).childAt(0).contains('Avatar')).toBeTruthy();
        expect(component.childAt(0).childAt(1).contains('Id')).toBeTruthy();
        expect(component.childAt(0).childAt(2).contains('Name')).toBeTruthy();
        
        expect(component).toMatchSnapshot();
    });
});
