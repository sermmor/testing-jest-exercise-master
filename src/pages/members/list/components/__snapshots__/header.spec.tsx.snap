// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/header test should be a valid header rendered 1`] = `
<thead>
  <tr>
    <th>
      Avatar
    </th>
    <th>
      Id
    </th>
    <th>
      Name
    </th>
  </tr>
</thead>
`;
